File: 01-Ejercicios-Clases/C4-08-25-23/Ejercicio01/principales/Principal.java
package principales;
import matematicas.*;

class Principal{
	public static void main(String[] args){
		Operacion op1 = new Operacion();
		int a = 10;
		int b = 20;
		System.out.println("La suma de " + a + " " + b + " es: " + op1.sumar(a,b));
	}
}

File: 01-Ejercicios-Clases/C5-08-29-23/Ejercicio02/Principal.cs
//using matematicas;
namespace principales{
	public class Principal{
		public static void Main(){
			System.Console.WriteLine("Metodo main...");
			matematicas.Operacion op = new matematicas.Operacion();
			int a = 10;
			int b = 20;
			System.Console.WriteLine("La suma de " + a + " y " + b + " es: " + op.sumar(a,b));
		}
	}
}